analyzer-comments/javascript/elyses-analytic-enchantments/must_avoid_imperative_loop.md

```diff
@@ -0,0 +1,7 @@
+# must avoid imperative loop
+
+Avoid use of for loops with this practice exercise.
+
+According to the instructions, there are build-in methods on the JavaScript Array global object that are geared towards analysis.
+
+Take a look at the instructions and/or the MDN docs on what methods are available to use.
```

Comment:
If its possible to include links here I would recommend to make MDN docs a link to the correct page.